diagnostics-app/src/app/telemetry/telemetry-card/telemetry-card.component.spec.ts

```diff
@@ -0,0 +1,23 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+
+import { TelemetryCardComponent } from './telemetry-card.component';
+import { TelemetryInfoType } from '@common/message';
+
+describe('TelemetryCardComponent', () => {
+  let component: TelemetryCardComponent;
+  let fixture: ComponentFixture<TelemetryCardComponent>;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      declarations: [TelemetryCardComponent]
+    });
+    fixture = TestBed.createComponent(TelemetryCardComponent);
+    component = fixture.componentInstance;
+    component.type = TelemetryInfoType.BATTERY;
+    fixture.detectChanges();
+  });
+
+  it('should create', () => {
```

Comment:
Consider adding some test case for error occurred

Reply:
For the scope of the intern project and with limited timeframe, I think think we will keep it to just feature development.

Unit test will be refactored and added by me in another time.